pkg/deploy/tagging/provider.go

```diff
@@ -53,6 +57,13 @@ func (p *defaultProvider) ResourceTags(stack core.Stack, res core.Resource, addi
 	return algorithm.MergeStringMap(stackTags, resourceIDTags, additionalTags)
 }
 
+func (p *defaultProvider) StackLabels(stack core.Stack) map[string]string {
+	normalizedStackID := strings.ReplaceAll(stack.StackID(), "/", "_")
```

Comment:
what is the reason behind normalization? The "/" character is also allowed in the tag value.

Reply:
it's not allowed as k8s label value :D. so i use stackLabels to identify k8s resources(targetGroupBinding) for a stack.